src/devices/machine/scc68070.h

```diff
@@ -29,14 +29,17 @@
 #include "cpu/m68000/m68000.h"
 
 
-#define ISR_MST     0x80    // Master
-#define ISR_TRX     0x40    // Transmitter
-#define ISR_BB      0x20    // Busy
-#define ISR_PIN     0x10    // No Pending Interrupt
-#define ISR_AL      0x08    // Arbitration Lost
-#define ISR_AAS     0x04    // Addressed As Slave
-#define ISR_AD0     0x02    // Address Zero
-#define ISR_LRB     0x01    // Last Received Bit
+#define ISR_MST         0x80                        // Master
+#define ISR_TRX         0x40                        // Transmitter
+#define ISR_BB          0x20                        // Busy
+#define ISR_PIN         0x10                        // No Pending Interrupt
+#define ISR_AL          0x08                        // Arbitration Lost
+#define ISR_AAS         0x04                        // Addressed As Slave
+#define ISR_AD0         0x02                        // Address Zero
+#define ISR_LRB         0x01                        // Last Received Bit
+#define	ISR_SSR_MASK    (ISR_MST | ISR_TRX | ISR_BB)// Mask for detecting start/stop/restart
+#define	ISR_START       (ISR_MST | ISR_TRX | ISR_BB)// Start bit request
+#define	ISR_STOP        (ISR_MST | ISR_TRX)			// Stop bit request
```

Comment:
The original header has *lots* of defines but they all look as though they should only ever bee used with scc68070.cpp so should probably be moved out of the header ?

Reply:
Yes, if they don’t need to be in the header, they shouldn’t be there.  Also consider using `static constexpr` or `enum` for constants rather than macros.  Macros give more confusing errors when things go wrong, and they don’t provide as much information for debuggers.